cmd/main.go

```diff
@@ -161,7 +161,8 @@ func run(c *cli.Context) error {
 	}
 
 	rpcCfg := &rpc.Config{
-		Port:    port,
+		Ctx:     ctx,
+		Address: fmt.Sprintf("127.0.0.1:%d", port),
```

Comment:
Unlike `swapd`, we are going to want the ability to listen on `0.0.0.0` or some specific external IP. We may want to change `--rpc-port` to something like `--rpc-bind-addr IP:PORT`. Should we move this `main.go` file into a subfolder like `cmd/swap-realayd`? (That was just the first name that came to my head for the executable, not saying it is ideal.)